pom.xml

```diff
@@ -330,6 +330,34 @@
             <ignoredUnusedDeclaredDependencies>org.objenesis:objenesis</ignoredUnusedDeclaredDependencies>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-enforcer-plugin</artifactId>
+          <version>3.0.0-M2</version>
+          <dependencies>
+            <dependency>
+              <groupId>com.google.cloud.tools</groupId>
+              <artifactId>linkage-checker-enforcer-rules</artifactId>
+              <version>1.0.0</version>
+            </dependency>
+          </dependencies>
```

Comment:
If we want this across the board, let's move it to the [shared-config parent](https://github.com/googleapis/java-shared-config) (common build rules for all the split repos) after we figure out the failure.